File: client_app/android/app/src/main/java/com/aionsigma_app/receivers/BootCompletedReceiver.java
package com.aionsigma_app.receivers;
import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import com.aionsigma_app.services.SyncDataService;

public class BootCompletedReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        if ("android.intent.action.BOOT_COMPLETED".equals(intent.getAction())) {
            Intent pushIntent = new Intent(context, SyncDataService.class);
            context.startService(pushIntent);
        }
    }
}

File: android/AionSigma/app/src/main/java/com/aionsigma/android/Receivers/BootCompletedReceiver.kt
package com.aionsigma.android.Receivers

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

import com.aionsigma.android.Services.SyncDataService


class BootCompletedReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
//        if ("android.intent.action.BOOT_COMPLETED" == intent.action) {
//            val pushIntent = Intent(context, SyncDataService::class.java)
//            context.startService(pushIntent)
//        }
    }
}